src/tetrominos.js

```diff
@@ -0,0 +1,80 @@
+// Ideally this would have been an enum if I were using TypeScript
+const TetrominoColours = {
```

Comment:
This is NOT being exported. Seems like it should be contained inside the file itself. Alternatively, once I start messing with `styled`, I could move this somewhere else...